packages/paste-core/components/truncate/src/index.tsx

```diff
@@ -13,6 +13,8 @@ const Truncate = React.forwardRef<HTMLSpanElement, TruncateProps>(({children, ..
       {...props}
       as="span"
       display="inline-block"
+      verticalAlign="bottom"
+      textDecoration="inherit"
```

Comment:
I still don't know if this is technically the  "fix" because the problem really is, that inline-block is adding some vertical space to the parent.

So if I place this in a heading for example, and change the color of the background so you can see the physical space the heading takes up, inline-block makes the whole thing bigger.

display block or flex corrects this, but that seems like it could be a risk change.

<img width="376" alt="image" src="https://user-images.githubusercontent.com/368249/174872481-a6a0df85-082a-482c-9b15-6a1080c8f5c6.png">

Reply:
Yeah... I thought about adding a `display` prop but I didn't want to change the default from `'inline-block'`. Even if we go that route, I still recommend keeping the vertical align because it does help the alignment when its inline block.